tests/src/utils/gameStateUtil.test.ts

```diff
@@ -13,7 +13,7 @@ testSingletonModule('utils/gameStateUtil', ({ testUnit }) => {
         sinon.stub(databaseUtil, 'hasGameFile').returns(true);
         const mockGameState = generateGameState();
         sinon.stub(databaseUtil, 'load').returns(mockGameState);
-        sinon.stub(databaseUtil, 'save');
+        sinon.stub(gameStateUtil, 'save');
```

Comment:
`gameStateUtil.save` is an independently testable unit, so it's a valid stopping point for this test.